cstar/system/environment.py

```diff
@@ -59,11 +59,23 @@ def __init__(
         mpi_exec_prefix: str,
         compiler: str,
     ):
-        # Initialize private attributes
+        """Initialize the instance.
+
+        Parameters
+        ----------
+        system_name : str
+            The name of the hosting platform
+        mpi_exec_prefix : str
+            The MPI prefix
+        compiler : str
+            The compiler to use for builds
+        """
         self._system_name = system_name
         self._mpi_exec_prefix = mpi_exec_prefix
         self._compiler = compiler
+        self._package_root: Path = self._find_package_root()
```

Comment:
```suggestion
        self._PACKAGE_ROOT: Path = self._find_package_root()
```

Consider renaming to be consistent w/other constant.

Reply:
Renamed